docs/releases/1.26-NOTES.md

```diff
@@ -63,6 +63,8 @@ error is encountered while updating an InstanceGroup.
 
 * Support for Kubernetes version 1.20 has been removed.
 
+* containerd config is now written to `/etc/containerd/config.toml`.
```

Comment:
I don't really consider this breaking change. It's not like we support operators interfering directly with this file. I'd move this to "other changes of note".

Reply:
True, they should not be messing with it.